testing/execution.ts

```diff
@@ -6,6 +6,7 @@ import {FormulaType} from '../runtime/types';
 import type {GenericSyncFormulaResult} from '../api';
 import type {MetadataContext} from '../api';
 import type {MetadataFormula} from '../api';
+import {Buffer as NonNativeBuffer} from 'buffer/';
```

Comment:
The trailing slash here is intentional: https://www.npmjs.com/package/buffer#usage

Reply:
TIL